Tests/colorLib/builder_test.py

```diff
@@ -1017,6 +1198,32 @@ def test_build_layerv1list_with_overlaps():
     assert colr.table.LayerV1List.LayerCount == 11
 
 
+def test_explicit_version_0():
```

Comment:
there are already test_explicit_version_0 and version_1 below, see same named methods in `BuildCOLRTest`

Reply:
Ugh yes. I think I deleted BuildCOLRTest at one point and "saved" that. Ofc now I brought it back :)